python/pecos/reps/pypmir/pypmir.py

```diff
@@ -25,6 +27,20 @@
 
 TypeOp = TypeVar("TypeOp", bound=op.Op)
 
+signed_data_types = {
+    "i8": np.int8,
+    "i16": np.int16,
+    "i32": np.int32,
+    "i64": np.int64,
+}
+
+unsigned_data_types = {
+    "u8": np.uint8,
+    "u16": np.uint16,
+    "u32": np.uint32,
+    "u64": np.uint64,
+}
+
```

Comment:
Unsure if this is the best module to place these data types.

Reply:
It's probably fine for now. I have been thinking some refactoring of PyPMIR might be good. So more thought can probably be put into a future PR.